frontend/src/App.jsx

```diff
@@ -24,32 +23,28 @@ export default function App() {
   // Components
   const loggedInRouter = (
     <RecoilRoot>
-    <BrowserRouter>
-      <Navbar onLogout={() => Auth.logout()} />
-
-      <div className="main-container">
-        <Switch>
-          <Route path="/posts">
-            <PostsPage />
-          </Route>
-
-          <Route path="/chat">
-            <ChatPage />
-          </Route>
-
-          <Route exact path="/">
-            <HomePage />
-          </Route>
-          <React.Suspense fallback={<div>Loading...</div>}>
-          <Route path="/profile">
-            <ProfilePage />
-          </Route>
-          </React.Suspense>
-        </Switch>
-      </div>
-
-      <Footer />
-    </BrowserRouter>
+      <BrowserRouter>
+        <Navbar onLogout={() => Auth.logout()} />
+
+        <div className="main-container">
+          <Switch>
+            <React.Suspense fallback={<div>Loading...</div>}>
+              <Route path="/posts">
+                <PostsPage />
+              </Route>
+              <Route path="/chat">
```

Comment:
I suggest we remove the chat here, as it's not yet implemented yet

Reply:
good idea